moveit_core/constraint_samplers/CMakeLists.txt

```diff
@@ -9,40 +9,43 @@ add_library(${MOVEIT_LIB_NAME}
 )
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
-target_link_libraries(${MOVEIT_LIB_NAME}
+
+ament_target_dependencies(${MOVEIT_LIB_NAME}
   moveit_robot_state
   moveit_kinematic_constraints
   moveit_kinematics_base
   moveit_planning_scene
-  ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+  urdf
+  urdfdom
+  urdfdom_headers
+  visualization_msgs)
 
 install(TARGETS ${MOVEIT_LIB_NAME}
-        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-        ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
-
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
-
-if(CATKIN_ENABLE_TESTING)
-  find_package(orocos_kdl REQUIRED)
-  find_package(angles REQUIRED)
-  find_package(tf2_kdl REQUIRED)
-  find_package(moveit_resources REQUIRED)
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
-include_directories(${orocos_kdl_INCLUDE_DIRS} ${angles_INCLUDE_DIRS} ${tf2_kdl_INCLUDE_DIRS} ${moveit_resources_INCLUDE_DIRS})
+install(DIRECTORY include/ DESTINATION include)
 
-  catkin_add_gtest(test_constraint_samplers
-    test/test_constraint_samplers.cpp
-    test/pr2_arm_kinematics_plugin.cpp
-    test/pr2_arm_ik.cpp
-  )
-  target_link_libraries(test_constraint_samplers
-    moveit_test_utils
-    ${MOVEIT_LIB_NAME}
-    ${catkin_LIBRARIES}
-    ${angles_LIBRARIES}
-    ${orocos_kdl_LIBRARIES}
-    ${urdfdom_LIBRARIES}
-    ${urdfdom_headers_LIBRARIES}
-  )
-endif()
+# if(BUILD_TESTING)
```

Comment:
I think it's fine to uncomment tests and just let them fail for now so we can fix them later.

Reply:
@vmayoral Let's get the test fixes cherry picked in

Reply:
It's already uncommented. 

https://github.com/AcutronicRobotics/moveit2/blob/constraint-samplers/moveit_core/constraint_samplers/CMakeLists.txt#L36-L81